CODEOWNERS

```diff
@@ -28,123 +28,3 @@
 # is not properly picked up in Github.
 * @cilium/janitors
 /api/ @cilium/api
-/api/v1/flow/ @cilium/api @cilium/hubble
-/api/v1/observer/ @cilium/api @cilium/hubble
-/api/v1/peer/ @cilium/api @cilium/hubble
-/bpf/ @cilium/bpf
-Makefile* @cilium/build
-/bpf/Makefile* @cilium/build @cilium/bpf
-/bpf/sockops/Makefile* @cilium/build @cilium/bpf
-Dockerfile* @cilium/build
-/bugtool/cmd/ @cilium/cli
-/cilium/ @cilium/cli
-/cilium/cmd/preflight_k8s_valid_cnp.go @cilium/kubernetes
-/cilium-health/ @cilium/health
-/cilium-health/cmd/ @cilium/health @cilium/cli
-/contrib/packaging/ @cilium/build
-/contrib/vagrant/ @cilium/contributing
-/contrib/coccinelle/ @cilium/bpf
-/daemon/ @cilium/agent
-/daemon/cmd/datapath.* @cilium/bpf
-/daemon/cmd/endpoint.* @cilium/endpoint
-/daemon/cmd/health.* @cilium/health
-/daemon/cmd/hubble.go @cilium/hubble
-/daemon/cmd/ipcache.* @cilium/ipcache
-/daemon/cmd/loadbalancer.* @cilium/loadbalancer
-/daemon/cmd/metrics.* @cilium/metrics
-/daemon/cmd/policy.* @cilium/policy
-/daemon/cmd/prefilter.go @cilium/bpf
-/daemon/cmd/proxy.go @cilium/proxy
-/daemon/cmd/state.go @cilium/endpoint
-/daemon/cmd/sysctl_linux.go @cilium/bpf
-/Documentation/ @cilium/docs
-/Documentation/cmdref @cilium/nonexistantteam
-/Documentation/bpf.rst @cilium/bpf
-/Documentation/contributing/ @cilium/contributing
-/Documentation/envoy/ @cilium/proxy
-/envoy/ @cilium/proxy
-/examples/ @cilium/docs
-/examples/kubernetes/ @cilium/kubernetes
-/examples/minikube/ @cilium/kubernetes
-*.Jenkinsfile @cilium/ci
-/hubble-relay/ @cilium/hubble
-/hubble-relay.Dockerfile @cilium/hubble
-/install/kubernetes/ @cilium/kubernetes @cilium/docs @cilium/helm
-jenkinsfiles @cilium/ci
-Jenkinsfile.nightly @cilium/ci
-/operator/ @cilium/operator
-/pkg/annotation @cilium/kubernetes
-/pkg/api/ @cilium/api
```

Comment:
The v1.7 equivalent commit  fd3b9ace2b278af74629d7767d71ae112b75b8cf leaves all of the @cilium/api codeowners in place, we should probably do the same here?